tests/test_requirements.txt

```diff
@@ -5,7 +5,6 @@ fastcache~=1.0.2
 glueviz~=0.13.3
 h5py~=3.1.0
 ipython~=7.6.1
-matplotlib~=3.3
```

Comment:
Please don't remove that. Change it to `matplotlib<3.4.0` instead.

Reply:
Fine for now, but FYI I'm trying to move yt to a better place in terms of dependency specifications, so I'll be working on this later.

Reply:
@neutrinoceros Before you dig any deeper on that, can we have a video conference about it so that (at least) I can understand it?  I'm anxious about moving too much since the pytest stuff is startign to get pulled in, and we don't want to have to move stuff on jenkins etc around too much just yet.

Reply:
Sure !